docs/UserGuide.md

```diff
@@ -664,64 +785,73 @@ VendorVault automatically saves your data as a JSON file `[JAR file location]/da
 
-## Known issues
+# 6 Known issues
 
 1. **When using multiple screens**, if you move the application to a secondary screen, and later switch to using only the primary screen, the GUI will open off-screen. The remedy is to delete the `preferences.json` file created by the application before running the application again.
 2. **If you minimize the Help Window** and then run the `help` command (or use the `Help` menu, or the keyboard shortcut `F1`) again, the original Help Window will remain minimized, and no new Help Window will appear. The remedy is to manually restore the minimized Help Window.
 
+[Back to Table of Contents](#table-of-contents)
+
 --------------------------------------------------------------------------------------------------------------------
 
-## Command Summary
+# 7 Command Summary
 
-### General Commands
+## 7.1 General Commands
 Action     | Format, Examples
 -----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------
-**[List](#viewing-all-deliveries-and-suppliers-list)**   | `list -a`
-**[Clear](#clearing-all-suppliers-and-deliveries-clear)**   | `clear`
-**[Help](#viewing-help-help)**   | `help`
-**[Exit](#exiting-the-program-exit)**   | `exit`
+**[List]()**   | `list -a`
+**[Clear]()**   | `clear`
+**[Help]()**   | `help`
+**[Exit]()**   | `exit`
 
 
-### Supplier-specific Commands
+## 7.2 Supplier-specific Commands
 
 Action     | Format, Examples
 -----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------
-**[Add](#adding-a-supplier-add-s)**    | `add -s n/NAME p/PHONE e/EMAIL com/COMPANY [t/TAG]…​ [pro/PRODUCT]…​` <br> e.g., `add -s n/John Doe p/98765432 e/johnd@example.com com/companyA t/friends t/owesMoney pro/rice pro/bread`
-**[Delete](#deleting-a-supplier-delete-s)** | `delete -s INDEX`<br> e.g., `delete -s 3`
-**[List](#listing-all-suppliers-list-s)**   | `list -s`
-**[Mark](#mark-a-supplier-with-a-status-mark-s)**   | `mark -s INDEX STATUS`<br> e.g.,`mark -s 2 active`
-**[Find](#find-a-supplier-find-s)**   | `find -s n/KEYWORD_FOR_SUPPLIER NAME> com/KEYWORD_FOR_SUPPLIER_COMPANY pro/<KEYWORD_FOR_SUPPLIER_PRODUCT>` <br> e.g., `find -s n/link com/NU`
-**[Sort](#sort-suppliers-sort-s)**   | `sort -s so/SORT_ORDER sb/SORT_BY_FIELD`<br> e.g., `sort -s so/a sb/n`
+**[Add]()**    | `add -s n/NAME p/PHONE e/EMAIL com/COMPANY [t/TAG]…​ [pro/PRODUCT]…​` <br> e.g., `add -s n/John Doe p/98765432 e/johnd@example.com com/companyA t/friends t/owesMoney pro/rice pro/bread`
+**[Delete]()** | `delete -s INDEX`<br> e.g., `delete -s 3`
+**[List]()**   | `list -s`
+**[Mark]()**   | `mark -s INDEX STATUS`<br> e.g.,`mark -s 2 active`
+**[Find]()**   | `find -s n/KEYWORD_FOR_SUPPLIER NAME> com/KEYWORD_FOR_SUPPLIER_COMPANY pro/<KEYWORD_FOR_SUPPLIER_PRODUCT>` <br> e.g., `find -s n/link com/NU`
```

Comment:
Should this follow the actual format listed for find -s command above?

Reply:
The command summary I will update once all commands are completed